Packs/CommonScripts/pack_metadata.json

```diff
@@ -2,7 +2,7 @@
     "name": "Common Scripts",
     "description": "Frequently used scripts pack.",
     "support": "xsoar",
-    "currentVersion": "1.2.56",
+    "currentVersion": "1.2.57",
```

Comment:
a new version of commonScripts exists. Please update currentVersion to `1.2.58`
Also, please remove you rn from version 1.2.57 to a new file `Packs/CommonScripts/ReleaseNotes/1_2_58.md`
You can do that by merging from master and running ` demisto-sdk update-release-notes -i Packs/CommonScripts -u revision`

Reply:
I did this, but now the version is up to 1.2.61, so the build is failing. Do you want me to update the release notes to 1.2.62 now?

I tried using update-release-notes but it didn't detect any changes, not sure why (possibly because I did a pull->merge with accept theirs instead of rebase).

Let me know.

Cheers
Adam

Reply:
Ok, I think I've got this now. I re-ran the command and it detected the changes and updated the pack medata accordingly. This should be resolved shortly.